ca/src/main/java/com/cletus/app/QueryProcessor.java

```diff
@@ -0,0 +1,37 @@
+package com.cletus.app;
+
+import java.io.*;
+import java.util.*;
+
+public class QueryProcessor {
+
+    public static String stripStopWords(String lang, String query) {
```

Comment:
Might want to think about how you would handle situtations with lowercase and uppercase mixed together (for the stop words).

Here's an example:  The dog aNd the cat -> dog cat

Reply:
Handled in the new code. Code only converts to lower case internally to check correctness, does not affect the output.